.github/CODEOWNERS

```diff
@@ -2,4 +2,4 @@
 # last matching pattern is used, see
 # https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners#codeowners-syntax
 
-* @oss-review-toolkit/bosch
+* @bs-ondem @MarcelBochtler @mnonnenmacher @nnobelis @oheger-bosch
```

Comment:
Would it make to create teams already that can be referred here? Like "frontend" / "backend", or "kotlin-devs" / "typescript-devs"?

Reply:
I cannot create teams on my own, so I'd rather go with those names first until it's clarified what options we have for creating teams, e.g. if non-committers could be added to teams without write permission.